stack.yaml

```diff
@@ -1,8 +1,12 @@
 flags: {}
 extra-package-dbs: []
+allow-newer: true
 packages:
 - .
 extra-deps:
```

Comment:
Might be this thing I added causing the problem. Was playing around with `stack` earlier and accidentally pushed my local version. Sorry about that!

Reply:
Ok, I'll remove this line when I merge my branch into master (unless you think it's better to keep it)